src/Corax/IndexWriter.cs

```diff
@@ -2024,6 +2026,8 @@ public void InsertTextualField()
                         postListIds = postListIds[idx..];
                         pageOffsets = pageOffsets[idx..];
                         entriesOffsets = entriesOffsets[idx..];
+                        sortedTerms = sortedTerms[idx..];
+                        termsOffsets = termsOffsets[idx..];
                     }
```

Comment:
Since now we are not updating the `sortedTerms` and `termsOffset` individually, shouldnt the update to those variables happening at https://github.com/ravendb/ravendb/blob/5f7c96bec8cb99d0773c9eaac9b9dca883f4d6bd/src/Corax/IndexWriter.cs#L1971 to be synchronized in the other references too?

Reply:
All of those values are being initialized in 
https://github.com/ravendb/ravendb/blob/5f7c96bec8cb99d0773c9eaac9b9dca883f4d6bd/src/Corax/IndexWriter.cs#L1980

So they are already synced.